README.md

```
@@ -50,6 +50,26 @@ dotnet run --project src/Worky.Cli -- scan --query '"backend engineer" hiring -i
 Default query targets hiring phrases in English, excluding replies and reposts.
 Output ranks matched posts by signal score, then recency, with the reason for each match.
 
+When `~/.worky/state.json` exists, both scan modes add a `network match` reason and a
++1.0 signal bonus to posts authored by anyone you follow. A missing or corrupt snapshot
+simply leaves scores untouched.
+
+### Targeted scan (follow graph)
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)